tools/sctrace/README.md

```diff
@@ -0,0 +1,357 @@
+# Syscall-Compliance-Trace
+
+Syscall-Compliance-Trace (sctrace) is a powerful system call compliance
+verification tool that analyzes and validates system call against user-defined
+compliance patterns. Written in SCML (System Call Matching Language), these
```

Comment:
Make "SCML (System Call Matching Language)" a hyperlink to the SCML section in the Asterinas book.